torchgeo/datasets/idtrees.py

```diff
@@ -211,10 +211,20 @@ def __getitem__(self, index: int) -> Dict[str, Tensor]:
         if self.split == "test":
             if self.task == "task2":
                 sample["boxes"] = self._load_boxes(path)
+                w, h = sample["image"].shape[1:]
+                sample["boxes"], _ = filter_boxes(
+                    image_size=(h, w), boxes=sample["boxes"]
```

Comment:
Doesn't matter if it's square, but should it both be `h, w`?

```suggestion
                h, w = sample["image"].shape[1:]
                sample["boxes"], _ = filter_boxes(
                    image_size=(h, w), boxes=sample["boxes"]
```

Reply:
Fixed